src/App.js

```diff
@@ -5,15 +5,25 @@ import Header from './components/Header/Header';
 import Footer from './components/Footer/Footer';
 import CentrePanel from './components/Centre/CentrePanel';
 
+import { BrowserRouter } from "react-router-dom";
```

Comment:
Could we use 1 import? All good otherwise

Reply:
I will update this, thanks